idem_azurerm/states/azurerm/postgresql/server.py

```diff
@@ -232,7 +234,7 @@ async def present(
         if login:
             ret["changes"]["new"]["administrator_login"] = login
         if login_password:
-            ret["changes"]["new"]["administrator_login_password"] = "REDACTED"
+            ret["changes"]["new"]["administrator_login_password"] = "REDACTED"  # nosec
```

Comment:
Is this necessary?

Reply:
Yup. That tells bandit to stop complaining about a hard-coded password... because it's not.